include/sch/Matrix/SCH_Types.h

```diff
@@ -35,6 +35,7 @@ namespace sch
   typedef  CD_Matrix::Vector3T<Scalar,true> Vector3NormOptimized;
 
   static const Scalar infinity = DBL_MAX;
+  static const Scalar epsilon = 1e-24;
```

Comment:
I think `std::numeric_limits<Scalar>::epsilon()` is a better value

Reply:
1. We are not sure that `std::numeric_limits<Scalar>::epsilon()` is defined, it depends on the type of Scalar. We can force the user to set if they have an exotic scalar though.
2. The value for the usual double is 2.2204460492503131e-16 which is huge. I, actually want to replace the preprocessor-defined values of epsilon (in https://github.com/jrl-umi3218/sch-core/blob/bf3c1892e402297b27796c267cce20da1dc4d87d/src/CD/CD_Pair.cpp#L20 and https://github.com/jrl-umi3218/sch-core/blob/bf3c1892e402297b27796c267cce20da1dc4d87d/src/CD_Penetration/CD_Depth.cpp#L6) with this one. I use it in some areas to check if the square of a norm is zero, which means the norm itself would be around 1e-8. That's huge.

What if I change the name of this variable?

Reply:
oops, my bad then :-)